demo/src/androidTest/java/com/novoda/movies/MovieItemViewTalkBackTest.java

```diff
@@ -34,8 +33,10 @@
     @Rule
     public MockitoRule mockitoRule = MockitoJUnit.rule();
 
+    private ViewTestRule<MovieItemView> viewTestRule = new ViewTestRule<>(R.layout.test_movie_item_view);
+
     @Rule
-    public ViewTestRule<MovieItemView> viewTestRule = new TalkBackViewTestRule<>(R.layout.test_movie_item_view);
+    public RuleChain chain = RuleChain.outerRule(new TalkBackTestRule()).around(viewTestRule);
```

Comment:
this is how to compose. they each (potentially) have a start and end, so we want the TalkBack one to be on the outside: this means talkback will start first, then the viewtestrule will run (start..end), and then back to the talkback one to end.